config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   root "home#index"
-  resources :home, only: [:index, :edit, :update]
+  resources :home, only: [:index, :edit, :update, :create]
```

Comment:
Are you saying that the form where a company signs up was hitting the home controller create action and not the companies controller create action?